exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -39,6 +39,7 @@
   String ZK_TIMEOUT = "drill.exec.zk.timeout";
   String ZK_ROOT = "drill.exec.zk.root";
   String ZK_REFRESH = "drill.exec.zk.refresh";
+  String ZK_SECURE_ACL = "drill.exec.zk.use.secure_acl";
```

Comment:
I was going to make this "zk.use_secure_acl" but when I was looking for an example I found "bit.use.ip", so I modeled it on that. I will change this to "zk.apply_secure_acl"